docs/en/concepts-and-designs/scope-definitions.md

```diff
@@ -414,3 +414,80 @@ This calculates the endpoint dimensional metrics data from each request of MQ sy
 | transmissionLatency | The latency from produce side to consume side .              |           | int(in ms) |
 | status              | Indicates the success or failure of the request.             |           | boolean    |
 | operation           | Indicates this access is on `Produce` or `Consume` side      |           | enum       |
+
+### SCOPES with `K8S` Prefix
+
+All metrics starting with `K8S` are derived from monitoring the Access Log in Kubernetes.
+
+#### Service, Service Instance and relations
+
+For all `K8SService`, `K8SInstance`, `K8SServiceRelation` and `K8SServiceInstanceRelation`, they all have the 
```

Comment:
I think you missed those scopes' own field. Same for endpoint and endpoint relation.

Reply:
Updated.